modules/apps/login/login-web/src/main/resources/META-INF/resources/login.jsp

```diff
@@ -219,6 +219,13 @@
 					});
 				}
 			}
+			AUI().ready(function (A) {
+				let signInButton = document.getElementsByClassName(
```

Comment:
any reason why here is `let` and not `const` as below?

Reply:
I think it was changed to const by SF, so I'll add const to both of them

Reply:
Hey @jgarcialr if it's not possible to use one single handler as the 2 modules are separate, let's at least try to avoid using AUI and resort to native JS, and wrap also this file to a separate JS resource that's then imported in the JSP file.

Reply:
Hi @gianmarcobrunialti ,
I used AUI as advised in a comment in the LPP by a FEI member. I can change it to native JS. 
Sorry I'm not a frontender :sweat_smile: so I'm not sure if this can be done in a single handler for both modules.

Reply:
If there's any example in other part of portal code that you're aware of, I can check it and apply same pattern

Reply:
Good like this then, if FEI is aware and advised this strategy (though controversial) let's keep the AUI().ready approach.

Reply:
Hi @gianmarcobrunialti ,
If there's a better approach not using the ready or if AUI will be deprecated / removed, I'm ok to change it to use plain js, using document.ready instead of the AUI ready